compose-tests/src/androidTest/java/radiography/test/compose/ComposeUiTest.kt

```diff
@@ -393,8 +396,8 @@ class ComposeUiTest {
 
     assertThat(hierarchy).isEqualTo(
       """
-      |CompositionLocalProvider:
-      |${BLANK}CompositionLocalProvider { test-tag:"parent" }
+      |Box:
```

Comment:
I'm not sure where this change comes from exactly, but it seems to be more correct. I am guessing it is a behavior change within compose that provides more accurate naming for nodes some how. I didn't look into it further though